### DISCUSSION TITLE
How to attach USB inside the vm?

### DISCUSSION THREAD

**Discussion by lattice0** [lima-vm/lima#2164] [Q&A]
I wear trying to get an USB inside my Linux vm on Linux host. AI only gives me hallucinations. I tried USB via IP, and in the host or was easy to setup but in the guest it requires building stuff, but I wanted a fast disposable VM. Is there a way to redirect USB devices just by command line when launching three lima vm?

**Comment by AkihiroSuda** [comment]
There is a PR, but not merged:
- https://github.com/lima-vm/lima/pull/1317
